### PR TITLE
[a11y] Ensure ARIA roles, states, and properties are valid

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
We made several changes to the Customizable Rollups configuration page to make it more accessible.
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
Tracking [W-7965362](https://gus.lightning.force.com/a07B0000008UEclIAG)

File: src/aura/CRLP_Rollup/CRLP_Rollup.cmp [hunk 1]
```diff
@@ -145,7 +145,6 @@
                                                     name="{!v.labels.description}"
                                                     value="{!v.activeRollup.description}"
                                                     readonly="{!v.isReadOnly}"
-                                                    required="{!v.isReadOnly ? 'false' : 'true'}"
```
Comment (lparrott) [inline src/aura/CRLP_Rollup/CRLP_Rollup.cmp:148]: Are these just unnecessary?

File: src/aura/CRLP_Rollup/CRLP_RollupHelper.js [hunk 2]
```diff
@@ -1413,14 +1413,27 @@
...
         //show help message in each child component SelectField
         if (!canSave) {
-            this.sendMessage(cmp, 'validateCmp');
+            this.sendMessage(cmp, 'validateCmp', cmp.get("v.labels.labelMissingRequired"));
         }
 
         //description and intValue set separately since we have direct access to these
         if (!activeRollup.description) {
-            cmp.find("descriptionInput").showHelpMessageIfInvalid();
+            cmp.find("descriptionInput").focus();
```
Comment (lparrott) [inline src/aura/CRLP_Rollup/CRLP_RollupHelper.js:1421]: This works great!

**Review by lparrott — COMMENTED** [review]
This works well, and the code looks good, I don't see any mention of `aria-live="assertive"` from the WI though, is that not needed?

**Pull request closed without merging**